src/libraries/System.Text.Json/src/System/Text/Json/Serialization/IJsonOnDeserialized.cs

```diff
@@ -0,0 +1,20 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json.Serialization
+{
+    /// <summary>
+    /// Specifies that the JSON object should have its <see cref="OnDeserialized"/> method called
```

Comment:
```suggestion
    /// Specifies that the type should have its <see cref="OnDeserialized"/> method called
```

Reply:
It is both the JSON (the "{" in JSON that represent JSON objects) and the CLR type (not a known value type and not an IEnumerable) that determines the behavior as to whether the default custom converter will be called that supports these notifications. I'll push another pass on these comments. Thanks